iris_grib/_grib_cf_map.py

```diff
@@ -116,8 +115,8 @@
     G2Param(2, 0, 6, 6): CFName('atmosphere_mass_content_of_cloud_liquid_water', None, 'kg m-2'),
     G2Param(2, 0, 6, 7): CFName('cloud_area_fraction_in_atmosphere_layer', None, '%'),
     G2Param(2, 0, 6, 25): CFName(None, 'WAFC_CB_horizontal_extent', '1'),
-    G2Param(2, 0, 6, 26): CFName(None, 'WAFC_ICAO_height_at_CB_base', 'm'),
-    G2Param(2, 0, 6, 27): CFName(None, 'WAFC_ICAO_height_at_CB_top', 'm'),
+    G2Param(2, 0, 6, 26): CFName(None, 'WAFC_ICAO_height_at_cloud_base', 'm'),
+    G2Param(2, 0, 6, 27): CFName(None, 'WAFC_ICAO_height_at_cloud_top', 'm'),
```

Comment:
@marqh This is a breaking change to the user, right?

Reply:
Hello @bjlittle 

this is an update, highlighting a mistake in the previous entries.

The term CB in WMO vocabularies is explicitly Cumulonimbus.
http://codes.wmo.int/ui/text-search?query=CB

These entries refer to 'convective cloud base'
http://codes.wmo.int/grib2/codeflag/4.2/0-6-26
which was interpreted by a domain expert as cloud base, but not Cumulonimbus.

This appears to be a semantic correction to me.
I don't infer from that that it is a 'breaking change'

It provides a consistency between the GRIB2 to CF Name and the CF Name to GRIB2 conversions which was previously missing.

Reply:
@marqh Thanks for the clarification :+1: